### DISCUSSION TITLE
希望新增火山翻译，腾讯翻译，自定义翻译引擎

### DISCUSSION THREAD

**Discussion by aegena** [YiiGuxing/TranslationPlugin#5732] [Ideas]
希望新火山翻译，腾讯翻译引擎
对于个人开发者而言阿里翻译现在是限制总额度，并没有百度，火山，腾讯的友好，
同时新增自定义翻译引擎能力，可以自有高度定置自己想要的服务器

**Comment by YiiGuxing** [comment]
嗯，这些都是有计划的，只是需要时间
